client/ayon_equalizer/addon.py

```diff
@@ -1,28 +1,43 @@
+"""3DEqualizer Addon for AYON.
+
+Addon definition for 3DEqualizer host in AYON.
+"""
 import os
+from typing import Any
+
 from ayon_core.addon import AYONAddon, IHostAddon
-from .version import __version__
 
+from .version import __version__
 
 EQUALIZER_HOST_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
 class EqualizerAddon(AYONAddon, IHostAddon):
+    """3DEqualizer Addon for AYON."""
+
     name = "equalizer"
     host_name = "equalizer"
-    heartbeat = 500
     version = __version__
+    heartbeat = 500
 
-    def initialize(self, settings):
+    def initialize(self, settings: dict[str, Any]) -> None:
+        """Initialize Equalizer Addon."""
         self.heartbeat = settings.get("heartbeat_interval", 500)
         self.enabled = True
 
-    def add_implementation_envs(self, env, _app):
-        # 3dEqualizer utilize TDE4_ROOT for its root directory
-        # and PYTHON_CUSTOM_SCRIPTS_3DE4 as a colon separated list of
-        # directories to look for additional python scripts.
-        # (Windows: list is separated by semicolons).
-        # Ad
+    def add_implementation_envs(self, env: dict, _app: str) -> None:
```

Comment:
It is `Application` from `ayon_applications.defs`, not `str`.
```suggestion
    def add_implementation_envs(self, env: dict, _app: Any) -> None:
```